src/starcheck.pl

```diff
@@ -331,8 +331,9 @@
 
 Ska::Starcheck::Obsid::set_config($config_ref);
 
-
-
+# Set the multple star filter disabled in the model if after this date
+my $MSF_ENABLED = $bs[0]->{time} > datetime('2016:102:00:00:00.000');
+set_acq_model_ms_filter($MSF_ENABLED);
```

Comment:
I don't have an easy way to set thi by keyword (pass ms_enabled=True/False) from Perl without using a wrapper, so is it acceptable to just set the value of ms enabled via arg directly like this?  Will that be an API problem later if you want to set other fit pars? 

Regarding the actual argument passed, there's also an Inline::Python object that can pass True/False, but that didn't seem to add any value over the 0/1 that goes through here.